alibi_detect/cd/tests/test_learned_kernel.py

```diff
@@ -37,7 +41,16 @@ def forward(self, x: torch.Tensor, y: torch.Tensor) -> torch.Tensor:
         return torch.einsum('ji,ki->jk', self.dense(x), self.dense(y))
 
 
-tests_lkdrift = ['tensorflow', 'pytorch', 'PyToRcH', 'mxnet']
+if has_keops:
+    class MyKernelKeops(nn.Module):
```

Comment:
Same comment as [this one](https://github.com/SeldonIO/alibi-detect/pull/602/files/1f6d5e02e6c89bec080874cd9fb3b02a3f9fdf28#r964683557)?